resources/views/form-right.blade.php

```diff
@@ -0,0 +1,50 @@
+<div class="card">
+    <div class="card-header" style="height: 62px">
+        <i class="fa fa-check"></i>{{'{{'}} trans('brackets/admin-ui::admin.forms.publish') }}
+    </div>
+    <div class="card-block">
+
+        <div class="form-group row align-items-center" :class="{'has-danger': errors.has('published_at'), 'has-success': this.fields.published_at && this.fields.published_at.valid }">
+            <label for="published_at" class="col-form-label text-md-right" :class="isFormLocalized ? 'col-md-2' : 'col-md-4'">{{'{{'}} trans('admin.{{ $modelVariableName }}.columns.published_at') }}</label>
+            <div :class="isFormLocalized ? 'col-md-4' : 'col-md-9 col-xl-8'">
+                <div class="input-group input-group--custom">
+                    <div class="input-group-addon"><i class="fa fa-calendar"></i></div>
+                    <datetime v-model="form.published_at" :config="datetimePickerConfig" v-validate="'date_format:yyyy-MM-dd HH:mm:ss'" class="flatpickr" :class="{'form-control-danger': errors.has('published_at'), 'form-control-success': this.fields.published_at && this.fields.published_at.valid}" id="published_at" name="published_at" placeholder="{{ trans('brackets/admin-ui::admin.forms.select_date_and_time') }}"></datetime>
+                </div>
+                <div v-if="errors.has('published_at')" class="form-control-feedback form-text" v-cloak>{{'@'}}{{'{{'}}errors.first('published_at') }}</div>
+            </div>
+        </div>
+    </div>
+</div>
+@if(in_array("created_by_admin_user_id", array_column($columns->toArray(), 'name')) && in_array("updated_by_admin_user_id", array_column($columns->toArray(), 'name')))
```

Comment:
As you see, also other columns are here, so it is not based on publishable, but can be also two columns if we have these fields.